web/src/routes/TuNewsDetailPage.tsx

```diff
@@ -42,8 +42,8 @@ const StyledTitle = styled.div`
   height: 100%;
   align-items: center;
   justify-content: center;
-  background-color: ${({ theme }) => theme.colors.tunewsThemeColor};
-  color: ${({ theme }) => theme.colors.backgroundColor};
+  background-color: ${props => props.theme.colors.tunewsThemeColor};
+  color: ${props => props.theme.colors.backgroundColor};
```

Comment:
i'm fine with having everything same style but the object destructuring is the proper way in this case? Even shorter

Reply:
Didn't work with stylelint, feel free to try though. But I don't see the benefit right now to change this everywhere.
Sure it is shorter, but why is one the proper way and the other not?